src/test/java/org/scalameter/examples/JavaRegressionTest3.java

```diff
@@ -21,7 +22,7 @@ public Persistor javaPersistor() {
 			  .put("exec.outliers.covMultiplier", 1.5)
 			  .put("exec.outliers.suspectPercent", 40);
 			public JavaGenerator<LinkedList<Integer>> generator() {
-				JavaGenerator<Integer> sizes = new SingleGen("size", 5000000);
+				JavaGenerator<Integer> sizes = new SingleGen("size", 5000000, ManifestFactory$.MODULE$.Int());
```

Comment:
I would suggest improving `japi` to have custom `*SingleGen` versions for `Int` and `Double`.